packages/react-components/src/docs/foundations/Typography.stories.mdx

```diff
@@ -2,6 +2,10 @@ import { Meta, Canvas, ArgsTable } from '@storybook/addon-docs';
 
 import { Heading, Text, Display } from '../../components/Typography';
 
+import { HeadingExamples } from '../components/Typography/HeadingExamples';
+import { TextExamples } from '../components/Typography/TextExamples';
+import { DisplayExamples } from '../components/Typography/DisplayExamples';
+
```

Comment:
Can you please move it into separate folder inside the Typography folder -> `stories` for example, so it would not mix up with normal components.